pkg/util/mount/mount_linux_test.go

```diff
@@ -151,3 +151,63 @@ func setEquivalent(set1, set2 []string) bool {
 	}
 	return true
 }
+
+func TestGetDevicePrefixRefCount(t *testing.T) {
+	fm := &FakeMounter{
+		MountPoints: []MountPoint{
+			{Device: "/dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2014-12.server:storage.target01-lun-1",
```

Comment:
For the sake of the test, can you please turn these strings into something shorter?  /mnt/111, /mnt/222, /mnt/333 or something that my puny brain can latch onto, please